docs/community-resources.md

```diff
@@ -7,22 +7,26 @@ sidebar_label: Community Resources
 Members of the PayID community have created useful resources for using and learning about PayID.
 
 **Tutorials**
-* [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
+
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
 
 **Validators**
-* [PayID Validator](https://payidvalidator.com/) - An [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
+
+- [PayID Validator](https://payidvalidator.com/) - An [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
 
   - [HTTP response headers](payid-headers)
   - [Cache-Control](payid-best-practices#cache-control)
   - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
   - Response time
 
-* [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator) - Validates URI strings to determine if they are valid PayIDs.
+- [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator) - Validates URI strings to determine if they are valid PayIDs.
 
 **Reference server implementations**
```

Comment:
```suggestion
**PayID server implementations**
```

I don't think we should call these "reference servers". Only our server at `payid-org/payid` is the reference server. These are just other PayID server implementations.